ui/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "mobile-security-service-ui",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
@damienomurchu should we be bumping the version? I haven't been doing this 🙈 If we need to can you post it in the channel.

Reply:
I think we need to, and realised we hadn't been doing so :)

Reply:
As discussed in standup today, we don't need to bump the version here but may bump the version as part of the overall component.